react/src/App.jsx

```diff
@@ -98,6 +101,10 @@ function App() {
     };
   }, []);
 
+  // @NOTE: We do not recommend adding events dynamically (on button clicks). Instead, we prefer to add them.
```

Comment:
no need for full stop after "them."
Add full stops only when the sentence ends - not at the end of every line.